tokio-console/src/state/tasks.rs

```diff
@@ -195,9 +195,9 @@ impl TasksState {
                 let id = ids.id_for(span_id);
 
                 let short_desc = strings.string(match (task_id, name.as_ref()) {
-                    (Some(task_id), Some(name)) => format!("{} ({})", task_id, name),
-                    (Some(task_id), None) => format!("{}", task_id),
-                    (None, Some(name)) => format!("({})", name),
+                    (Some(task_id), Some(name)) => format!("{task_id} ({name})"),
+                    (Some(task_id), None) => task_id.to_string(),
+                    (None, Some(name)) => name.to_owned()
```

Comment:
missing `,` here (i think that was my fault, sorry!)
```suggestion
                    (None, Some(name)) => name.to_owned(),
```

Reply:
Yep, rustc caught that one for me, I didn't notice it either. (-;

I needed to use `name.as_ref().to_owned()` here because `name` is an `InternedStr`. Not sure if there's a better pattern for this.

Reply:
Oh, whoops, yeah, that makes sense. I assumed it was just a string.